numpy/core/tests/test_umath.py

```diff
@@ -1759,7 +1759,15 @@ def test_reduce(self):
 
 class TestBitwiseUFuncs:
 
-    bitwise_types = [np.dtype(c) for c in '?' + 'bBhHiIlLqQ' + 'O']
+    _all_ints_bits = [
+        np.dtype(c).itemsize * 8 for c in np.typecodes["AllInteger"]]
+    bitwise_types = [
+        np.dtype(c) for c in '?' + np.typecodes["AllInteger"] + 'O']
+    bitwise_bits = [
+        2,  # boolean type
```

Comment:
`2` feels a bit confusing for boolean, but I guess its fine.

Reply:
This line `assert i == np.bit_count(input_dtype(num)), msg`  converts `input_dtype(<non zero>)` to `True`, hence bitcount stays `1`, so I added 2.